src/shared_modules/content_manager/src/components/factoryDownloader.hpp

```diff
@@ -54,6 +78,12 @@ class FactoryDownloader final
         {
             return std::make_shared<S3Downloader>();
         }
+        if (downloaderType.compare("offline") == 0)
```

Comment:
Question, any reason for not doing
```suggestion
        if (downloaderType == "offline")
```
?

Reply:
I just did it like in the rest of the class.

Changed!